packages/order-watcher/src/schemas/websocket_schemas.ts

```diff
@@ -0,0 +1,32 @@
+export const webSocketUtf8MessageSchema = {
+    id: '/webSocketUtf8MessageSchema',
+    properties: {
+        utf8Data: { type: 'string' },
+    },
+    type: 'object',
+    required: ['utf8Data'],
+};
+
+export const webSocketRequestSchema = {
+    id: '/webSocketRequestSchema',
+    properties: {
+        action: { enum: ['GET_STATS', 'ADD_ORDER', 'REMOVE_ORDER'] },
+        signedOrder: { $ref: '/signedOrderSchema' },
+        orderHash: { type: 'string' },
```

Comment:
Let's use `hexSchema` here, it's stricter.

Reply:
cool. that's exactly what I was looking for.